main.go

```diff
@@ -278,6 +282,9 @@ func runWithOptions(opts Options, out output) {
 	}
 
 	if opts.TOpts.CallerName != "" {
+		if val, ok := disallowedCallerNames[opts.TOpts.CallerName]; ok {
+			out.Fatalf("Disallowed caller name: %v", val)
```

Comment:
This will print "Disallowed caller name: true". You meant to use `opts.TOpts.CallerName`.